packages/docusaurus-module-type-aliases/src/index.d.ts

```diff
@@ -140,17 +140,17 @@ declare module '@docusaurus/Head' {
   import type {HelmetProps} from 'react-helmet-async';
   import type {ReactNode} from 'react';
 
-  export type HeadProps = HelmetProps & {children: ReactNode};
+  export type Props = HelmetProps & {children: ReactNode};
```

Comment:
🤔 it's not an interface 😄

Reply:
Mmm, this is core API and doesn't get either wrapped or swizzled anyways, so I think it's fine